nestkernel/connection_manager.h

```diff
@@ -393,6 +400,17 @@ ConnectionManager::get_max_delay() const
   return max_delay_;
 }
 
+inline void
+ConnectionManager::append_move_connections_( std::deque< ConnectionID >& input,
```

Comment:
I, personally, prefer to implement it as a static inline method in cpp file (local for cpp). But `grep` gave me just one place in the code where this approach is used. So I decided that this is not common practice for nest and made it as a class member.
Do you think it's ok to declare this method as static and place it just before `get_connections` in cpp file? Or do you think that moving implementation to cpp and leaving declaration in hpp is better approach?